studio/src/aem/mas-filter-toolbar.js

```diff
@@ -19,7 +20,7 @@ class MasFilterToolbar extends LitElement {
     `;
 
     static properties = {
-        searchText: { type: String, state: true },
+        searchText: { type: String, state: true, attribute: 'search-text' },
```

Comment:
do we need it as an attribute?  As you user changes the search text, we would end up having an inconsistency with that is in the attribute and what is in the search. If we use `reflect:true`, that it will cause unnecessary DOM updates.

Reply:
this is used for init